packages/block-library/src/template-part/edit/index.js

```diff
@@ -97,21 +84,17 @@ export default function TemplatePartEdit( {
 			// FIXME: @wordpress/block-library should not depend on @wordpress/editor.
 			// Blocks can be loaded into a *non-post* block editor.
 			// eslint-disable-next-line @wordpress/data-no-store-string-literals
-			const defaultWrapperElement = select( 'core/editor' )
+			const areaObject = select( 'core/editor' )
 				.__experimentalGetDefaultTemplatePartAreas()
-				.find( ( { area: value } ) => value === _area )?.area_tag;
+				.find( ( { area: value } ) => value === _area );
```

Comment:
Since we are getting this info here, I think we can assign a default areaObject in `js` and loose the `areaObject?.area_tag ?? 'div'` type of checks and also pass this info to `TemplatePartPlaceholder` which does a similar call for these.

Reply:
> pass this info to TemplatePartPlaceholder which does a similar call for these.

I don't like passing props down when you can access them with useSelect. It can be a performance improvement when we do noticed issues but otherwise, I prefer to design the API of a component based on semantics, not based on what info is computed above or what data is rendered.